test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -40,7 +40,7 @@ def step(context):
         _statusChat.send_message(row[0])
         _statusChat.verify_last_message_sent(row[0])
 
-@Then("user is able to send  a random chat message")
+@Then("user is able to send a random chat message")
```

Comment:
We agreed on this so you could, as well, refactor this!
```suggestion
@Then("the user is able to send a random chat message")
```